libs/coin-modules/coin-evm/src/api/index.ts

```diff
@@ -61,9 +62,8 @@ export function createApi(config: EvmConfig, currencyId: CryptoCurrencyId): Api
     getBlockInfo(_height: number): Promise<BlockInfo> {
       throw new Error("getBlockInfo is not supported");
     },
-    getStakes(_address: string, _cursor?: Cursor): Promise<Page<Stake>> {
-      throw new Error("getStakes is not supported");
-    },
+    getStakes: (address: string, cursor?: Cursor): Promise<Page<Stake>> =>
+      getStakes(currency, address, cursor),
```

Comment:
Implementations don't have to support all parameters, especially when unused 
```suggestion
    getStakes: (address: string): Promise<Page<Stake>> =>
      getStakes(currency, address),
```

Reply:
OK ! Thought it was necessary for readability
Removed !